### DISCUSSION TITLE
大概需要多少数据进行训练，才能达到不错的效果

### DISCUSSION THREAD

**Discussion by xielongzhu** [PaddlePaddle/PaddleOCR#12383] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by xielongzhu** [comment]
在标注自己的数据集时，对文字的倾斜度有要求吗？，比如30°、60°、90°，180°的要给正过来吗？是必须保持文字水平并且正方向的吗，有大佬回答一下吗

**Comment by xielongzhu** [comment]
标注一行数据，其中有间隔性的空格，该如何标注？

**Comment by tink2123** [comment, marked as answer]
文字尽量保持水平正向，一行数据中多个空格，可以合并成一个。

文本识别的数据量取决于字典大小，平均字典中每个字需要出现200次。 文本检测数据量推荐至少在1000张左右

**Comment by TOMORINAONAO** [comment]
> 文字尽量保持水平正向，一行数据中多个空格，可以合并成一个。
> 
> 文本识别的数据量取决于字典大小，平均字典中每个字需要出现200次。 文本检测数据量推荐至少在1000张左右

这个每个字出现200次，如果我是用的手写汉字数据集，一个单字可能有50张左右不同的写法，那最终的训练数据中是这个单字出现200次以上还是200*50=10000次以上？
